portfolio/src/main/webapp/photography.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+        <nav>
+            <a href="index.html">ABOUT ME</a>
+            <a href="projects.html">PROJECTS</a>
+            <a class="active" href="/">PHOTOS</a>
+        </nav>
+        <div class="content">
+
+            <h2>Photography</h2>
+            <p>I love photographing sunsets, sunrises and landscapes. There's just a feeling of peace and beauty when you see the nature around you.
+            I'm very much a nightowl and frequently see the sunrises and love to take the opportunity to take a picture and remember the moment. In addition
+            I love to visit places where you can get great views like mountains or the beach. <p>
+            <p>Click the button below to get a random picture of a sunset or sunrise I took!</p>
+            <button onclick="randomPicture()">Get a Random Picture!</button>
+            <div id="picture-container">
+                <img src="/images/Landscape1.jpg" alt="landscape" id="landscapeImage">
+            </div>
+        </div>
+        <footer>
+            <p> © Martin Heberling</p>
+        </footer>
+  </body>
+</html>
```

Comment:
nit: add the newline at the end of your files

Reply:
Will do.